contracts/gateway/EIP20CoGateway.sol

```diff
@@ -179,6 +178,12 @@ contract EIP20CoGateway is CoGateway {
 
     /* public variables */
 
+    /** address of utility token. */
+    address public utilityToken;
+
+    /** address of value token. */
+    address public valueToken;
```

Comment:
Why not of type interface? Do you use the imported `UtilityTokenInterface` anywhere?